.github/workflows/ci.yml

```diff
@@ -69,3 +69,7 @@ jobs:
           version: nightly
       - name: Run tests
         run: cargo test --release --all-targets
+      - name: Run solidity tests
+        run: |
+          cd ethereum
+          forge test
```

Comment:
Maybe as a separate step? It will parallelize workflow run

Reply:
It runs instantaneously, so not sure if it worth it.